hrpc/get.go

```diff
@@ -18,8 +19,41 @@ type Get struct {
 	baseQuery
 	// Don't return any KeyValue, just say whether the row key exists in the
 	// table or not.
-	existsOnly bool
-	skipbatch  bool
+	existsOnly  bool
+	skipbatch   bool
+	consistency QueryConsistency
+}
+
+// QueryConsistency is used to specify the required consistency of data
+//
+// See https://docs.cloudera.com/HDPDocuments/HDP2/HDP-2.2.9/bk_hadoop-ha/content/ha-hbase-timeline-consistency.html
+type QueryConsistency int
+
+const (
+	// Use HBase's default
+	DefaultConsistency QueryConsistency = iota
```

Comment:
The default is strong consistency, perhaps we could remove this?
```
	Default_Get_Consistency      = Consistency_STRONG
...
func (x *Get) GetConsistency() Consistency {
	if x != nil && x.Consistency != nil {
		return *x.Consistency
	}
	return Default_Get_Consistency
}
```

Reply:
Why would we have `DefaultConsistency` when HBase proto doesn't have one and if the user try to use this value, it will just panic. Better not have it at all no?

Reply:
> Setting it will not cause a panic
Ah yes sorry I misread that part. Make sense to me. Let's ship it!